### PR TITLE
emacs: new package at 26.3

### PR DESCRIPTION
**Pull request by penguin-ff**
## Description of package

GNU Emacs

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by dylanaraps** [comment]
LGTM :+1:

I'll land the package manager update first and will merge this afterwards.

File: community/emacs/sources [hunk 1]
```diff
@@ -0,0 +1 @@
+https://gnuftp.uib.no/emacs/emacs-26.3.tar.xz
```
Comment (dithpri) [inline community/emacs/sources]: Shouldn't this be either the [main GNU ftp](https://ftp.gnu.org/gnu/emacs/emacs-26.3.tar.xz) or an auto-redirect using http://ftpmirror.gnu.org/emacs/?
I don't think it makes sense to use one mirror for every KISS download.

Comment (penguin-ff) [reply community/emacs/sources]: You're right. I'll take "your" first link...

**Comment by dylanaraps** [comment]
Thanks :+1:

**Pull request merged**